### PR TITLE
Feature: Data query runs can also run events

### PR DESCRIPTION
**Pull request by akshaysasidrn**
Changes:
- Data query manager also renders event manager
- Data query run triggers for `onDataQuerySuccess` events upon success
- Data query run triggers for `onDataQueryFailure` event upon failure

Screenshots:
image
image
image

Loom: https://www.loom.com/share/ee4fe477c22a4ed2bcaddafdf2ada0b7

### CODE REVIEW

File: frontend/src/_styles/theme.scss [hunk 1]
```diff
@@ -2077,13 +2077,8 @@ input:focus-visible {
 
 input[type=text] {
   border-radius: 0 !important;
-  outline-color: #dadcde !important;
 }
 
-.widget-header {
```
Comment (gondar00) [inline frontend/src/_styles/theme.scss:2083]: Need this class for the widget sidebar header

File: frontend/src/_styles/theme.scss [hunk 2]
```diff
@@ -2077,13 +2077,8 @@ input:focus-visible {
 
 input[type=text] {
   border-radius: 0 !important;
-  outline-color: #dadcde !important;
```
Comment (gondar00) [inline frontend/src/_styles/theme.scss:2080]: need this for removing blue outline from certain i/p fields

File: frontend/src/_styles/theme.scss (hunk 1 above)
Comment (akshaysasidrn) [reply frontend/src/_styles/theme.scss:2083]: I see, will update

File: frontend/src/_styles/theme.scss (hunk 2 above)
Comment (Ckbhatia) [reply frontend/src/_styles/theme.scss:2080]: You folks should avoid putting `!important` wherever possible.

File: frontend/src/_styles/theme.scss (hunk 1 above)
Comment (akshaysasidrn) [reply frontend/src/_styles/theme.scss:2083]: Have reverted the changes. Was from merge conflict resolution.

**Pull request merged**